openshift-fluentd-syslog-journal.conf

```diff
@@ -0,0 +1,25 @@
+<source>
+  @type systemd
+  path "#{ENV['DATA_DIR']}/journal"
+  pos_file "#{ENV['DATA_DIR']}/journal.pos"
+  tag system.var.log.messages
```

Comment:
why system.var.log.messages - it's not from /var/log/messages? maybe it is better to make sure tag indicates that it is coming from systemd